errors/errorHandler.js

```diff
@@ -1,10 +1,19 @@
 const errorHandler = (err, req, res, next) => {
```

Comment:
This error handling needs looking at.
You can start extracting them into seperate error handlers, for custom thrown errors and PSQL errors for example.